device-types/Cisco/C9800-CL-K9.yaml

```diff
@@ -0,0 +1,8 @@
+---
+manufacturer: Cisco
+model: Catalyst 9800-CL-K9
+slug: cisco-c9800-cl-k9
+part_number: C9800-CL-K9
+u_height: 1
+is_full_depth: false
+comments: '[Cisco Catalyst 9800-CL Wireless Controller for Cloud Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9800-cl-wireless-controller-cloud/nb-06-cat9800-cl-cloud-wirel-data-sheet-ctp-en.html)'
```

Comment:
This is really a VM, not a device, it should not be included.

Reply:
removed